ocfweb/announcements/templates/announcements/2017-03-20-hiring.html

```diff
@@ -0,0 +1,121 @@
+    <div class="row">
+        <div class="col-sm-8 ocf-content-block">
+            <h3></h3>
+            <p><em>March 20, 2017</em></p>
+            <hr />
+            <p>
+                The OCF is making a big transition to being completely
+                self-staffed next semester, and as part of that process, we are
+                looking to hire students to be OCF Operations staff for the
+                2017-2018 academic year! This includes two paid positions,
+                Operations Assistant and Operations Manager, neither of which
+                require any technical experience. These are very important
+                roles, and without dedicated and passionate students to fill
+                them, the OCF will not be able to stay open next semester.
+            </p>
+
+            <h3>Operations Assistants</h3>
+            <p>
+                Operations Assistants will have the following responsibilities:
+
+                <ul>
+                    <li>Staffing the front desk</li>
+                    <ul>
+                        <li>Maintaining lab security</li>
+                        <li>Providing help to OCF members using the lab</li>
+                        <li>Carrying out regular lab maintenance</li>
+                    </ul>
+                    <li>Working on projects that benefit the OCF</li>
+                    <ul>
+                        <li>Marketing and outreach</li>
+                        <li>Collecting feedback on OCF services</li>
+                        <li>Organizing events taking place in the lab</li>
+                        <li>Researching and applying for additional sources of
+                            funding for the OCF</li>
+                    </ul>
+                    <li>Attending regular meetings with the Operations Manager
+                        and the other Operations Assistants</li>
+                </ul>
+
+                This job position will be paid $13.75 per hour, but will also be
+                less work and more flexible than the Operations Manager
+                position. See the
+                <a href="https://docs.google.com/document/d/1oS3ma415LbtuyeEuuoucWKYLcWOJaWmzhv2nIs5f718/edit">full job description</a>
+                for more details.
+            </p>
+
+            <h3>Operations Manager</h3>
+            <p>
+                The Operations Manager will have the same responsibilities as
+                the Operations Assistants (see above), except they will spend
+                less time staffing the front desk of the OCF, and instead will
+                spend the remaining time performing more managerial duties,
+                such as leading meetings with the Operations Assistants,
+                delegating projects, and scheduling shifts for the OCF front
+                desk. This job will involve more work than the Operations
+                Assistant role, but it is also paid more, at $15.75 per hour.
```

Comment:
Good point, those would be better to emphasize, it is more work in that they have to work 12 hours instead of a minimum of 8 for the OAs, but I do think emphasizing the responsibilities of the role would be a better idea.